docs/developer-networking.md

```diff
@@ -69,12 +70,34 @@ open dev.test.crt
 
 When prompted to add the certificate to your Keychain, choose the System keychain.  After verifying that you are an administrator, search for `dev.test` in Keychain Access and double click the certificate.  Open the Trust section and in the top drop-down select Always Trust.
 
-# Nginx Proxy
+# Traefix proxy
 
-Before you begin this, make sure you are in the root directory of the application.
+Update the Virtual-Machine repo's `.tools/traefik/config.yml` file to reference these certificates.
```

Comment:
@dtgreiner can you put an example `config.yml` here?  I don't think there was anything secret in that, so should be fine to include here, rather than referencing the GR only repo.